### PR TITLE
Update submodule and patchset

### PR DESCRIPTION
**Pull request by feiwan1**
Submodule ffmpeg 6b213175c9..271a0a55bc

Fix apply conflict of 0062
Fix build error of 0127

### CODE REVIEW

**Comment by xhaihao** [comment]
@feiwan1  There are errors in pre-check.

File: patches/0062-lavu-add-sub-frame-side-data.patch [hunk 1]
```diff
@@ -45,10 +45,18 @@ index 7828c94dc5..1201a08636 100644
...
-index b6cee2d886..db423990f0 100644
+index 930f01dc19..40c67de35b 100644
 --- a/libavutil/frame.c
 +++ b/libavutil/frame.c
-@@ -333,7 +333,9 @@ FF_ENABLE_DEPRECATION_WARNINGS
+@@ -56,6 +56,7 @@ static const AVSideDataDescriptor sd_props[] = {
+     [AV_FRAME_DATA_SPHERICAL]                   = { "Spherical Mapping",                            AV_SIDE_DATA_PROP_GLOBAL },
+     [AV_FRAME_DATA_ICC_PROFILE]                 = { "ICC profile",                                  AV_SIDE_DATA_PROP_GLOBAL },
+     [AV_FRAME_DATA_SEI_UNREGISTERED]            = { "H.26[45] User Data Unregistered SEI message",  AV_SIDE_DATA_PROP_MULTI },
++    [AV_FRAME_DATA_SUB_FRAME]                   = { "Sub frame Metadata" },
```
Comment (xhaihao) [inline patches/0062-lavu-add-sub-frame-side-data.patch:55]: Is a AV_SIDE_DATA_PROP_xxx needed ?

File: patches/0062-lavu-add-sub-frame-side-data.patch [hunk 2]
```diff
@@ -59,19 +67,19 @@ index b6cee2d886..db423990f0 100644
...
-     case AV_FRAME_DATA_DOVI_RPU_BUFFER:             return "Dolby Vision RPU Data";
-     case AV_FRAME_DATA_DOVI_METADATA:               return "Dolby Vision Metadata";
-     case AV_FRAME_DATA_AMBIENT_VIEWING_ENVIRONMENT: return "Ambient viewing environment";
-+    case AV_FRAME_DATA_SUB_FRAME:                   return "Sub frame Metadata";
-     }
+@@ -941,6 +944,7 @@ const AVSideDataDescriptor *av_frame_side_data_desc(enum AVFrameSideDataType typ
+     unsigned t = type;
+     if (t < FF_ARRAY_ELEMS(sd_props) && sd_props[t].name)
+         return &sd_props[t];
++
```
Comment (xhaihao) [inline patches/0062-lavu-add-sub-frame-side-data.patch:74]: Please remove the new line.

File: patches/0062-lavu-add-sub-frame-side-data.patch (hunk 1 above)
Comment (feiwan1) [reply patches/0062-lavu-add-sub-frame-side-data.patch:55]: No, just normal side data type can satisfy our request.

**Review by xhaihao — APPROVED** [review]
LGTM

**Comment by xhaihao** [comment]
The error in pre-check is not caused by this PR.

**Pull request merged**